### ISSUE TITLE
Need a better README.md

### ISSUE THREAD

**Issue by chpapa** [oursky/react-native-oursky#1]
To explain what the project is about so people can take a quick look and aware of what is it.

**Comment by Amdadullah-Raihan** [comment]
Hello Sir,

I am interested in contributing to improving the README.md. Is there any guideline to follow?

**Comment by chpapa** [comment]
@Amdadullah-Raihan Not really, but I think it would be really good if the README.md have a brief intro to Authgear (we can provide that), with a very simple quick start?

**Comment by Amdadullah-Raihan** [comment]
Please provide anything needed,  I would love to update the Readme.md file.
